android/app/src/main/java/app/organicmaps/bookmarks/Holders.java

```diff
@@ -183,27 +183,23 @@ protected void setSize()
       mSize.setText(getSizeString());
     }
 
-    private String getSizeString()
-    {
+    private String getSizeString() {
```

Comment:
1. It's better to avoid reformatting existing code.
2. It's better to follow the existing code style instead of mixing it.